pkg/sfu/streamallocator/streamallocator.go

```diff
@@ -555,7 +555,7 @@ func (s *StreamAllocator) processEvents() {
 }
 
 func (s *StreamAllocator) ping() {
-	ticker := time.NewTicker(500 * time.Millisecond)
+	ticker := time.NewTicker(100 * time.Millisecond)
```

Comment:
Doing this more frequently as all probe checking happens async via this. Want to ensure that probe finalise happens promptly.